eZ/Publish/Core/MVC/Symfony/Templating/Twig/FieldBlockRenderer.php

```diff
@@ -358,10 +305,50 @@ private function getRenderFieldBlockName($fieldTypeIdentifier, $type)
      *
      * @return string
      */
-    private function getRenderFieldDefinitionBlockName($fieldTypeIdentifier, $type)
+    private function getRenderFieldDefinitionBlockName($fieldTypeIdentifier, $type): string
     {
         $suffix = $type === self::EDIT ? self::FIELD_DEFINITION_EDIT_SUFFIX : self::FIELD_DEFINITION_VIEW_SUFFIX;
 
         return $fieldTypeIdentifier . $suffix;
     }
+
+    /**
+     * @param string $resourceType
+     *
+     * @return array
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     */
+    protected function getResources(string $resourceType): array
+    {
+        if ($resourceType === 'fieldViewResources') {
+            return $this->sortResources($this->resourceProvider->getFieldViewResources());
+        }
+
+        if ($resourceType === 'fieldEditResources') {
+            return $this->sortResources($this->resourceProvider->getFieldEditResources());
+        }
+
+        if ($resourceType === 'fieldDefinitionViewResources') {
+            return $this->sortResources($this->resourceProvider->getFieldDefinitionViewResources());
+        }
+
+        if ($resourceType === 'fieldDefinitionEditResources') {
+            return $this->sortResources($this->resourceProvider->getFieldDefinitionEditResources());
+        }
+
+        throw new InvalidArgumentException(
+            '$resourceType',
+            sprintf('Invalid resource type: %s', $resourceType)
+        );
+    }
+
+    protected function sortResources(array $resources): array
+    {
+        usort($resources, static function (array $a, array $b) {
```

Comment:
Does it return `int`? If so, maybe:
```suggestion
        usort($resources, static function (array $a, array $b): int {
```